docs/contivctl.md

```diff
@@ -0,0 +1,227 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+*Note:*
+`contivctl` Command can be found under `$GOPATH/bin` directory (`/opt/gopath/bin` in vagrant VMs) after a build. 
+Currently, `contivctl` needs to be run on the same host where `netmaster` is running
+
+## Usage
+
```

Comment:
@mapuri thats a good point.
@jainvipin 
This file is too detailed for an end user. Let me add a separate user facing document about contiv object model and how to use contivctl command. I'll structure it the way you mentioned above.
after that we can remove references to docs/ConfigDetails.md